app/requests_handlers.py

```diff
@@ -613,11 +626,35 @@ async def post(self, request):
                     fulfilment_code_array.append(available_fulfilments[0]['fulfilmentCode'])
 
                 try:
-                    await RHService.request_fulfilment_post(request,
-                                                            attributes['case_id'],
-                                                            attributes['first_name'],
-                                                            attributes['last_name'],
-                                                            fulfilment_code_array)
+                    room_number = attributes['roomNumber']
+                except KeyError:
+                    room_number = None
+
+                if room_number:
+                    if len(attributes['last_name']) < 48:
+                        last_name = attributes['last_name'] + ', ' + room_number
+                        title = None
+                    else:
+                        last_name = attributes['last_name']
+                        title = room_number
+                else:
+                    last_name = attributes['last_name']
```

Comment:
I've just checked the UI  manually and there doesn't appear to be any limit to the surname entered (I was able to type a surname 243 characters long for example).  When we try to post this we get a rejection at RHSvc and an ERROR log on RHUI, and a "Something went wrong" page.  Shouldn't we limit the surname to 60 characters at the UI to match the server ?  Perhaps using the 'charCheckLimit' that you had on the room name ? We might like to think about this for some other fields also.

Reply:
Will get a ticket raised to do cover this throughout RHUI